bitnami/jaeger/templates/collector/deployment.yaml

```diff
@@ -0,0 +1,182 @@
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.collector.podAntiAffinityPreset "component" "collector" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.collector.nodeAffinityPreset.type "key" .Values.collector.nodeAffinityPreset.key "values" .Values.collector.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.collector.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.collector.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.collector.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.collector.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.collector.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.collector.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.collector.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.collector.topologySpreadConstraints "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+      {{- include "jaeger.waitForDBInitContainer" . | nindent 8 }}
+      {{- if .Values.collector.initContainers }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.collector.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: jaeger-collector
+          image: {{ include "jaeger.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.collector.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.collector.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.collector.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.collector.command "context" $) | nindent 12 }}
+          {{- else }}
+          command: ["/opt/bitnami/jaeger/bin/jaeger-collector"]
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.collector.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.collector.args "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.collector.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.collector.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
```

Comment:
Does jaeger expose a `/metrics` endpoint in any of the ports? You may want to create a metrics service for scrapping these metrics

Reply:
Added metrics annotations for each service [1024807](https://github.com/bitnami/charts/pull/13480/commits/10248071e9a922ce6c787039b9cc2c751401f1b9)